x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/InferenceStats.java

```diff
@@ -62,19 +65,22 @@ public static String docId(String modelId, String nodeId) {
     private final long missingAllFieldsCount;
     private final long inferenceCount;
     private final long failureCount;
+    private final long cacheMissCount;
     private final String modelId;
     private final String nodeId;
     private final Instant timeStamp;
 
     private InferenceStats(Long missingAllFieldsCount,
                            Long inferenceCount,
                            Long failureCount,
+                           Long cacheMissCount,
                            String modelId,
                            String nodeId,
                            Instant instant) {
         this(unbox(missingAllFieldsCount),
             unbox(inferenceCount),
             unbox(failureCount),
+            unbox(cacheMissCount),
```

Comment:
I know it was already in this file, but, as above, maybe `unboxOrZero` would be clearer.